### PR TITLE
Better readme

### PR DESCRIPTION
**Pull request by lepieru**
Just a better readme page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,28 @@
-# nimbus-theme
-The best dark theme for Emacs
+# Nimbus theme for Emacs
 
-## Demos
+The best dark theme for Emacs.
 
-[Python](http://i.imgur.com/2Ij1ROs.png)
+## Screeshots
```
Comment (m-cat) [inline README.md]: Typo: should be "Screenshots"

File: README.md [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
 
-[Elisp](http://i.imgur.com/iSXPuqB.png)
+Python:
 
-[Nim](https://i.imgur.com/nPLOtwL.png)
+![Python](http://i.imgur.com/2Ij1ROs.png)
```
Comment (m-cat) [inline README.md]: I'm not sure about inlining the images, as they take up a lot of space and can be a pain to scroll past. Adding a table of contents might help, but the images would still autoload and be annoying for people. I would suggest inlining one image maximum, and keeping the rest as links. Thanks for the contribution!

Comment (lepieru) [inline README.md]: I agree. Done.

**Comment by m-cat** [comment]
Awesome!

**Pull request merged**